aztk/client/base/helpers/get_application_log.py

```diff
@@ -0,0 +1,114 @@
+import time
+
+import azure
+import azure.batch.models as batch_models
+import azure.batch.models.batch_error as batch_error
+
+from aztk import error
+from aztk import models
+from aztk.utils import constants, helpers
+
+output_file = constants.TASK_WORKING_DIR + \
```

Comment:
I'm actually thinking we shouldn't do path.join since this path is a Linux style path (evaluated on the node, not the client) regardless of what client runs it. So putting "/" explicitly seems better.

Reply:
yeah, sure. created a separate issue for that: #630